certified-connectors/DocuSign/apiDefinition.swagger.json

```diff
@@ -889,6 +890,221 @@
         "x-ms-visibility": "important"
       }
     },
+    "/accounts/salesCopilotAccount/envelopes/getRelatedActivities": {
```

Comment:
Hi @harshitav-docusign, 

Could you please fix swagger validation errors?  please refer the swagger schema file, this would help to fix the issue. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/apiDefinition.swagger.schema.json